### ISSUE TITLE
Mobile VoiceOver problem when "step" attribute is too small

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/sun#509]
Discovered while working on #499, and required for https://github.com/phetsims/gravity-force-lab-basics/issues/119. `<input type="range">` doesn't receive any input events from VoiceOver when the `step` attribute is too small relative to the range of the slider. In gravity force lab basics, the step attribute is 1/1000000000 of the range. If the `step` attribute is made larger to be 1/100000 of the range, input events are received. Currently, the `step` attribute is determined by 

```js

          // The step attribute must be non zero for the accessible input to receive all accessibility events, and
          // only certain values are allowed depending on the step size, or else the AT will announce values as
          // "Invalid". If the step size is equal to the precision of the slider readout, all values are allowed.
          this.setAccessibleAttribute( 'step', Math.pow( 10, -this.a11yDecimalPlaces ) );

```

This line was determined to be the best in https://github.com/phetsims/sun/issues/413 for non-mobile AT. We should return there for history.

**Comment by jessegreenberg** [comment]
This isn't working because the `step` is using different units than the physical Property in GFLB. `step` is in billions of kg, while the range is in kg.

**Comment by jessegreenberg** [comment]
I tried step="any" again (which is a valid DOM attribute) from #413, but it does't work with iOS VoiceOver either.

**Comment by jessegreenberg** [comment]
I workaround has been added with the above commit, along with a lot of documentation that describes PhET's slider conquest until this point. Closing this issue, since discussion can continue in the related https://github.com/phetsims/sun/issues/495
